gnucash/gnome-utils/account-quickfill.c

```diff
@@ -408,6 +442,22 @@ listen_for_account_events (QofInstance *entity, QofEventId event_type,
         DEBUG("other %s", name);
         break;
     }
+    // Now that qfb->list_store_full has been updated, qfb->list_store also needs to be updated in
+    // case we're using the regular search.
```

Comment:
We prefer to use `/* ... */` for multiline comments. And lest you object that it makes commenting out code hard, we prefer `#if 0 ... #endif` for that.